src/nwb_datajoint/common/common_waveforms.py

```diff
@@ -84,9 +91,8 @@ def load_waveforms(self, key: dict):
         we = si.WaveformExtractor.load_from_folder(we_path)
         return we
     
-    def fetch_nwb(self, key):
-        # TODO: implement fetching waveforms from NWB
-        return NotImplementedError
+    def fetch_nwb(self, *attrs, **kwargs):
+        return fetch_nwb(self, (AnalysisNwbfile, 'analysis_file_abs_path'), *attrs, **kwargs)
```

Comment:
has this been tested? I'm not confident about the way I saved the waveforms to NWB file...

Reply:
Not tested. The fetch call should work, but what it will return I can't say.